### DISCUSSION TITLE
typer: %E4%B8%80%E4%B8%AA%E9%9D%9E%E5%B8%B8%E5%A5%BD%E7%94%A8%E7%9A%84%E5%91%BD%E4%BB%A4%E8%A1%8C%E5%BA%94%E7%94%A8%E5%BA%93 | boringhex.top %E5%8D%9A%E5%AE%A2

### DISCUSSION THREAD

**Discussion by giscus** [boringhex-top/blog.boringhex.top#6] [Announcements]
# typer: %E4%B8%80%E4%B8%AA%E9%9D%9E%E5%B8%B8%E5%A5%BD%E7%94%A8%E7%9A%84%E5%91%BD%E4%BB%A4%E8%A1%8C%E5%BA%94%E7%94%A8%E5%BA%93 | boringhex.top %E5%8D%9A%E5%AE%A2

Typer, build great CLIs. Easy to code. Based on Python type hints.  今天偶然看到GitHub上一个python开源项目——Typer，可以帮助简单快速地构建命令行应用。正如介绍所说，“users will love using and developers will love creating”。  The key featur

https://blog.boringhex.top/posts/b94f0fda7898/

**Comment by pengwon** [comment]
非常好用
